Add Vanadium guide (#1978)
privacyguides/privacyguides.org: docs/mobile-browsers.md

[Brave Sync](https://support.brave.com/hc/articles/360059793111-Understanding-Brave-Sync) allows your browsing data (history, bookmarks, etc.) to be accessible on all your devices without requiring an account and protects it with E2EE.

### Vanadium (GrapheneOS)

<div class="admonition recommendation" markdown>

![Vanadium logo](assets/img/browsers/vanadium.svg#only-light) ![Vanadium logo](assets/img/browsers/vanadium-dark.svg#only-dark){ align=right }

> **Vanadium** is the default browser that is included with [GrapheneOS](../android.md#grapheneos), it includes a number of [patches](https://github.com/GrapheneOS/Vanadium/tree/main/patches) which assist in improving security and privacy of WebView.
> 
> Vanadium is built upon the Chromium web browser project, so it should feel familiar and have minimal website compatibility issues.

Reviewer: ```suggestion
**Vanadium** is the default browser that is included with [GrapheneOS](../android.md#grapheneos). The Chromium-based browser includes a number of [patches](https://github.com/GrapheneOS/Vanadium/tree/main/patches) which help improve the security and privacy of the WebView used by other apps.

Among the numerous privacy and security enhancements listed in their documentation, Vanadium notably includes a built-in content blocker and disables JavaScript Just-In-Time (JIT) compilation by default, both of which can be toggled on a per-site basis.
```

<details class="downloads" markdown>
<summary>Downloads</summary>

> - [:octicons-home-16: Homepage](https://vanadium.app){ .md-button .md-button--primary }
> - [:octicons-code-16:](https://github.com/GrapheneOS/Vanadium){ .card-link title="Source Code" }

Reviewer: ```suggestion
[:octicons-home-16: Homepage](https://vanadium.app){ .md-button .md-button--primary }
[:octicons-info-16:](https://grapheneos.org/usage#web-browsing){ .card-link title=Documentation }
[:octicons-code-16:](https://github.com/GrapheneOS/Vanadium){ .card-link title="Source Code" }
```

> <details class="downloads" markdown>
> <summary>Downloads</summary>
> 
> - [:octicons-home-16: Homepage](https://vanadium.app){ .md-button .md-button--primary }
> - [:octicons-code-16:](https://github.com/GrapheneOS/Vanadium){ .card-link title="Source Code" }
> 
> </details>

Reviewer: ```suggestion
- [:octicons-home-16: Homepage](https://vanadium.app){ .md-button .md-button--primary }
- [:octicons-code-16:](https://github.com/GrapheneOS/Vanadium){ .card-link title="Source Code" }
```

Reviewer: Vanadium is installed by default on GrapheneOS, so a "Downloads" toggle isn't needed here.

Reviewer 2: @all-contributors add @redoomed1 for review

> </div>

Reviewer: ```suggestion
</div>

#### Recommended Vanadium Configuration

<div class="annotate" markdown>

As explained in their documentation, Vanadium prioritizes better privacy- and security-enhancing defaults to avoid users making themselves stand out by opting into non-default settings.

You might, however, consider configuring the following privacy-related settings, all of which can be found in :material-dots-vertical: → **Settings** → **Privacy and security**.

- [x] Select **Close tabs on exit**
- [x] Select **Open external links in incognito** (1)
- [x] Select **Lock Incognito tabs when you leave Vanadium**

</div>

1. With this setting enabled, **all** links from other apps, custom tabs, search intents and share intents will open in incognito tabs.
```

Reviewer: > As explained in their documentation, Vanadium prioritizes better privacy- and security-enhancing defaults to avoid users making themselves stand out by opting into non-default settings.

I reference the paragraph starting with "Extension support isn't planned" on their [documentation](https://grapheneos.org/features#vanadium).

Mull comes with privacy protecting settings configured by default. You might consider configuring the **Delete browsing data on quit** options in Mull's settings if you want to close all your open tabs when quitting the app automatically, or clear other data such as browsing history and cookies automatically.

> #### Adblocking
> 
> Currently the only way to block ads with Vanadium is to set a [DNS Resolver](../dns.md) that has this functionality. The option for that can be found in :material-dots-vertical: → **Settings** → **Privacy and security**. If you're using a VPN we suggest using your VPN provider's server.
> 
> - [x] Select **Use secure DNS** > **Choose another provider**
> 

Reviewer: ```suggestion
```

Reviewer: I suggested basic formatting changes and other fixes first. I'll suggest description changes and recommended configuration later.
